Source/WebCore/css/html.css

```diff
@@ -860,9 +861,13 @@ input:is([type="button"], [type="submit"], [type="reset"]), input[type="file"]::
     padding: 0 1.0em;
     border: 1px solid -webkit-control-background;
     font: 11px system-ui;
-    background-color: -apple-system-opaque-secondary-fill;
     color: -apple-system-blue;
-#endif
+#if defined(WTF_PLATFORM_VISION) && WTF_PLATFORM_VISION
+    background-color: -apple-system-opaque-secondary-fill;
+#else
+    background-color: rgba(0, 0, 0, 0.05);
```

Comment:
Wouldn't it be the other way around? I might be misreading though

Reply:
yes